src/main/java/build/buildfarm/common/redis/RedisHashMap.java

```diff
@@ -45,6 +51,22 @@ public RedisHashMap(String name) {
     this.name = name;
   }
 
+  /**
+   * @brief Set key to hold the string value and set key to timeout after a given number of seconds.
+   * @details If the key already exists, then the value is replaced.
+   * @param jedis Jedis cluster client.
+   * @param key The name of the key.
+   * @param value The value for the key.
+   * @param expiration_s Timeout to expire the entry. (units: seconds (s))
+   * @return Whether a new key was inserted. If a key is overwritten with a new value, this would be
+   *     false.
+   * @note Overloaded.
+   */
+  @Override
+  public boolean insert(JedisCluster jedis, String key, String value, int expiration_s) {
+    return jedis.hset(name, key, value) == 1;
```

Comment:
And the unique timeout insert has its expiration ignored. Call the timeout ignorant insert here.